src/components/ErrorMesage/ErrorMesage.css

```diff
@@ -0,0 +1,38 @@
+div.notification {
+  position: relative;
+  color: rgba(195, 18, 18);
+  font-size: 24px;
+  background-color: #e7c1c1;
+  padding: 25px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)